cmd/queue/main.go

```diff
@@ -322,8 +333,51 @@ func main() {
 		logger.Fatalw("Failed to parse localhost URL", zap.Error(err))
 	}
 
+	queueProxyL3 := fmt.Sprintf("%s:%d", env.ServingPod, networking.ServiceHTTPPort)
+	zipkinEndpoint, err := zipkin.NewEndpoint(env.ServingPod, queueProxyL3)
+
+	if err != nil {
+		logger.Error("Unable to create tracing endpoint")
+		return
+	}
+
+	oct := tracing.NewOpenCensusTracer(
+		tracing.WithZipkinExporter(tracing.CreateZipkinReporter, zipkinEndpoint),
+	)
+	fmt.Println("TRACING VARS")
+	fmt.Println(os.Getenv("TRACING_CONFIG_ENABLE"))
+	fmt.Println(os.Getenv("TRACING_CONFIG_DEBUG"))
+	fmt.Println(os.Getenv("TRACING_CONFIG_SAMPLE_RATE"))
+	fmt.Println(os.Getenv("TRACING_CONFIG_ZIPKIN_ENDPOINT"))
+
+	enableVal, err := strconv.ParseBool(os.Getenv("TRACING_CONFIG_ENABLE"))
```

Comment:
Please use the envconfig at the top of the file. You can even nest it and have a specific struct with just the tracing config in it. All the defaulting and parsing can be handled there.

Reply:
Thanks! Just looked into the envconfig library - that's helpful.